blinkt.go

```diff
@@ -93,17 +119,39 @@ func (bl *Blinkt) Show() {
 }
 
 // SetAll sets all pixels to specified r, g, b colour. Show must be called to update the LEDs.
-func (bl *Blinkt) SetAll(r int, g int, b int) {
+func (bl *Blinkt) SetAll(r int, g int, b int, brightness ...float64) {
+
 	for i, _ := range bl.pixels {
 		bl.SetPixel(i, r, g, b)
 	}
+
+	if len(brightness) > 0 {
+		bl.SetBrightness(brightness[0])
+	}
+
 }
 
 // SetPixel sets an individual pixel to specified r, g, b colour. Show must be called to update the LEDs.
-func (bl *Blinkt) SetPixel(p int, r int, g int, b int) {
+func (bl *Blinkt) SetPixel(p int, r int, g int, b int, brightness ...float64) {
+
 	bl.pixels[p][0] = r
```

Comment:
If I've understood your suggestion correctly then yes, great idea, this would make the code more readable.

```
const RedIndex int = 0
const GreenIndex int = 1
const BlueIndex int = 2
const BrightnessIndex int = 3

bl.pixels[p][RedIndex ] = r
bl.pixels[p][GreenIndex ] = g
bl.pixels[p][BlueIndex ] = b
bl.pixels[p][BrightnessIndex ] = brightness[0]
```